src/cclib/writer/cjsonwriter.py

```diff
@@ -17,97 +17,424 @@
     has_openbabel = False
 
 import os.path
+import ntpath
 import json
+import numpy as np
 
 from . import filewriter
 
 
 class CJSON(filewriter.Writer):
     """A writer for chemical JSON (CJSON) files."""
+    
+    # The expected Key names for all supported attributes.
+    _attrkeynames = {
```

Comment:
This has been implemented in this [commit](https://github.com/cclib/cclib/pull/279/commits/b941191fcafe1b6ae37695eefff88c4d50fc3bec)
The data.py file already had a dictionary containing the attribute keys, to avoid redundant code the key:value dictionary has been converted to a key:namedtuple pair. The necessary changes have been made in all files